api.planx.uk/modules/send/idox/nexus.ts

```diff
@@ -172,14 +178,14 @@ async function checkUniformAuditTable(
 async function authenticate({
   clientId,
   clientSecret,
-}: UniformClient): Promise<UniformAuthResponse> {
+}: IdoxNexusClient): Promise<IdoxNexusAuthResponse> {
   const authString = Buffer.from(`${clientId}:${clientSecret}`).toString(
     "base64",
   );
 
   const authConfig: AxiosRequestConfig = {
     method: "POST",
-    url: process.env.UNIFORM_TOKEN_URL!,
+    url: process.env.IDOX_NEXUS_TOKEN_URL!,
```

Comment:
nit: We should maybe `assert()` on these variables instead of using `!` (this isn't unique or new to this PR I know).

I'd actually really like to parse the `.env` file to handle all this at some point, something like this - https://www.youtube.com/watch?v=q1im-hMlKhM

Reply:
They won't be set on production yet, only have staging values - hence why I'm not `assert()`-ing yet! Definitely one to adjust later once more stable

Reply:
Ah perfect, that makes sense 👍